### PR TITLE
Cummulative usability testing tweaks

### PR DESCRIPTION
**Pull request by vblagoje**
Various tweaks. @tristantarrant please review and adjust if needed.

### CODE REVIEW

File: src/main/webapp/cache-container/configuration-security/security.html [hunk 1]
```diff
@@ -80,7 +80,6 @@
         <div class="form-group">
           <div class="col-md-10 col-md-offset-2">
             <button type="button" class="btn btn-primary" ng-click="defineAuthorization()">Yes</button>
```
Comment (tristantarrant) [inline src/main/webapp/cache-container/configuration-security/security.html]: I prefer verbs on buttons. Therefore "Enable Authorization"

File: src/main/webapp/cluster-nodes/cluster-nodes.html [hunk 2]
```diff
@@ -72,10 +72,11 @@ <h2 class="card-pf-title">Add Node
             inetAddress:server.getInetAddress()})" title="View Node Details">
           <div class="card-pf card-pf-accented card-pf-aggregate-status card-pf-with-action">
             <h2 class="card-pf-title">
-              <span class="pficon" ng-class="server.isStopped()?'fa fa-power-off':'pficon-ok'"></span>{{server.name}}
+              <span class="pficon" ng-class="server.isStopped()?'fa fa-power-off':'pficon-ok'"></span>{{server.getHost()}}
```
Comment (tristantarrant) [inline src/main/webapp/cluster-nodes/cluster-nodes.html]: I think we should swap them: server name (which is unique across the cluster) in the card title and host name in the card body

Comment (vblagoje) [inline src/main/webapp/cluster-nodes/cluster-nodes.html]: Aha, yeah it makes sense. One sec

File: src/main/webapp/cache-container/configuration-security/security.html (hunk 1 above)
Comment (vblagoje) [inline src/main/webapp/cache-container/configuration-security/security.html]: Ok, correcting

**Comment by tristantarrant** [comment]
Thanks Vladimir, pushed with some additional changes.

**Pull request closed without merging**